### ISSUE TITLE
Warning in HA log: Entity "does not have device_class defined"

### ISSUE THREAD

**Issue by TheArcturian** [jseidl/hass-magic_areas#112]
I'm just wondering if this bug is getting fixed in a future release of Magic Areas. This is a short extract from the HA log and I got a lot of them:

`2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_screen_0_height does not have device_class defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_battery_remaining_time does not have device_class defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_network_0_bytes_sent does not have device_class defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_network_1_bytes_sent does not have device_class defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_storage_d_available_free_space does not have device_class defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_network_1_ipv6 does not have device_class defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.master_iphone_battery_state does not have unit_of_measurement defined
2021-04-13 09:18:37 WARNING (MainThread) [custom_components.magic_areas.sensor] Entity sensor.aeon_storage_d_format does not have device_class defined`

Not a big deal, just would be nice to clean up the logs so that other relevant warnings in the log is more visible.

**Comment by jseidl** [comment]
Yeah probably this needs to be downgraded to info or either debug.

**Comment by jseidl** [comment]
Being fixed for 3.0.0
